readthedocs_theme/templates/includes/topnav.html

```diff
@@ -1,13 +1,17 @@
 {# The main product menu, for all high priority information #}
 {% macro menu_product(with_more=False) %}
   <div class="header">About</div>
+  <a class="item" href="{{ SITEURL }}/features/">
+    <i class="fad fa-arrow-progress primary icon"></i>
+    How Read the Docs works
+  </a>
   <a class="item" href="{{ SITEURL }}/pricing/">
     <i class="fad fa-diagram-venn primary icon"></i>
     Pricing
   </a>
-  <a class="item" href="{{ SITEURL }}/features/">
-    <i class="fad fa-arrow-progress primary icon"></i>
-    How Read the Docs works
+  <a class="item" href="{{ SITEURL }}/pricing/enterprise/">
+    <i class="fad fa-diagram-venn primary icon"></i>
```

Comment:
This is the same icon as the pricing plans menu item, it should be something different.

![image](https://github.com/readthedocs/website/assets/1140183/aa3d7cb4-bfc3-4c7b-ab5c-a1b462e72e08)

Reply:
```suggestion
    <i class="fad fa-briefcase primary icon"></i>
```